public_html/script.js

```diff
@@ -60,6 +62,24 @@ function processReceiverUpdate(data) {
                 var ac = acs[j];
                 var hex = ac.hex;
                 var plane = null;
+		
```

Comment:
You should do this once per plane (when creating the PlaneObject below), not every time.

Also, preprocess the static data so it has the bits you need (e.g. "full country name" vs "path to image").